suite-native/app/e2e/pageObjects/settingsActions.ts

```diff
@@ -45,10 +45,13 @@ class SettingsActions {
         await waitFor(currencySelectorTriggerElement).toBeVisible().withTimeout(10000);
         await currencySelectorTriggerElement.tap();
 
+        await wait(1000); // wait for the currency selector to open
+
         const currencySelectorItemElement = element(by.id(`@select/item/${currencyCode}`));
         await scrollUntilVisible(currencySelectorItemElement, '@bottom-sheet/scroll-view');
 
         await currencySelectorItemElement.tap();
+        await wait(1000); // wait for the currency selector to close
```

Comment:
```suggestion
   await waitFor(currencySelectorTriggerElement).toBeVisible().withTimeout(10000);
```
I think this is safer

Reply:
same here